tron/kubernetes.py

```diff
@@ -135,8 +135,42 @@ def handle_event(self, event: Event) -> None:
         if k8s_type == "running":
             self.started()
         elif k8s_type == "finished":
-            self.exited(0)
+            raw_object = getattr(event, "raw", {})
+            container_statuses = raw_object.get("status", {}).get("container_statuses", [])
```

Comment:
Do we know if k8s will ever give us a `null` but existent`container_statuses` (i.e. the `get()` will give us `None` which will make our `len()` fail later)?  I think I've seen funky stuff like 
```
container_statuses = pod_status.get('status', {}).get('container_statuses')
container_statuses = container_statuses if container_statuses else []
```
in our code that means to save ourselves from those weird edge cases, but not sure it's necessary here.